src/Console/Locale/EnableLocaleCommand.php

```diff
@@ -219,6 +219,14 @@ private function askToInstall()
             return false;
         }
 
+        $this->formatter->writeln(
+            '<info>Before you can enable a new locale you need to authenticate to be able to create the pages</info>'
+        );
+
+        while (!Authentication::loginUser($this->formatter->ask('Login'), $this->formatter->askHidden('Password'))) {
```

Comment:
I agree with @StijnVrolijk since if you have access to this command you also can write custom code